terraform/rds/rds.tf

```diff
@@ -0,0 +1,48 @@
+resource "aws_rds_cluster" "rdsclu" {
+    cluster_identifier      = var.cluster_name
+    engine                  = "aurora-mysql"
+    engine_version          = "5.7.mysql_aurora.2.07.2"
+    db_subnet_group_name    = var.db_subnet_group_name
+    vpc_security_group_ids  = [aws_security_group.rds-apps.id]
+    availability_zones      = var.availability_zones
+    database_name           = var.db_name
+    master_username         = "admin"
+    master_password         = "admin" # the password will be repalced when security manager set up
+    backup_retention_period = 1
+    deletion_protection     = false
+    apply_immediately       = true
+    skip_final_snapshot     = true
+}
+
+resource "aws_rds_cluster_instance" "cluster_instances" {
+    count                  = 1
+    identifier             = "aurora-cluster-instance"
+    cluster_identifier     = aws_rds_cluster.rdsclu.id
+    instance_class         = "db.r4.large" # this is the smallest instance can be use for the engine
```

Comment:
you sure you can't use db.t3.small ?

Reply:
Yes, db.t3.small is not supported for the aurora db this engine version, got an error said the combinations of db.t3.small and 5.7.mysql_aurora.2.07.2 is not supported, so I checked the console of creating the db, and "db.r4.large" seems like the smallest available option.

Reply:
if that is too expensive might be just better to switch to RDS.. I will leave that with you, I am just suggesting and discussing, not imposing anything

Reply:
Sure, I just tested it and deleted it straight away, let's see.